be/package.json

```diff
@@ -8,7 +8,7 @@
     "dev:start": "nodemon",
     "debug": "DEBUG=express:* node dist/app.js",
     "test": "jest --passWithNoTests",
-    "lint": "eslint",
+    "lint": "eslint .",
```

Comment:
should this be `.` or rather `./src` ?

Reply:
The latter is better, changed and then merged.